README.md

```diff
@@ -0,0 +1,23 @@
+# Unnamed-Rails-Project
+
+- This project is about creating a token that people can use to have impact on websites such as blogs.
+
+[![Build Status](http://img.shields.io/travis/badges/badgerbadgerbadger.svg?style=flat-square)](https://travis-ci.org/badges/badgerbadgerbadger) [![License](http://img.shields.io/:license-mit-blue.svg?style=flat-square)](http://badges.mit-license.org)
```

Comment:
We don't use travis

Reply:
Now, we do 💃

Reply:
It is not merged yet. Build status in readme should be the part of travis PR